src/compile/unit.ts

```diff
@@ -249,27 +249,6 @@ export class UnitModel extends ModelWithField {
     return this.encoding;
   }
 
-  public toSpec(excludeConfig?: any, excludeData?: any) {
```

Comment:
Removed unused method.

Reply:
Are you sure it is unused across projects?

Reply:
I think compassql might use it but I think then it would make sense to move it there. https://github.com/search?q=org%3Avega+toSpec&type=Code